src/Material.cpp

```diff
@@ -279,9 +279,9 @@ FP_PRECISION* Material::getDifHat() {
 
   if (_dif_hat == NULL) {
 
-    _dif_hat = new FP_PRECISION[4*_num_groups];
+    _dif_hat = new FP_PRECISION[6*_num_groups];
 
-    for (int e = 0; e < 4*_num_groups; e++)
+    for (int e = 0; e < 6*_num_groups; e++)
```

Comment:
With this update, we are now solving CMFD in 3D. The z-height of the cmfd lattice is just required to be 1 and the z-max and z-min BCs are required to be reflective. This effectively makes the problem 2D.